api/users.go

```diff
@@ -248,6 +255,44 @@ func (r usersRoutes) addSingleUserToCourse(name string, email string, course mod
 	}
 }
 
+type pinRequest struct {
```

Comment:
Can you inline the definition of `pinReques` into `pinCourse`? It is not used outside that method, so we want to hide it.